packages/font-glyphs/src/letter/shared/shoulder-bowl.ptl

```diff
@@ -0,0 +1,244 @@
+			create-glyph name code : glyph-proc
+				local df : DivFrame 1
+				local ArcC1 : ac1 df.leftSB df.rightSB XH 0 (stroke -- df.mvs) (hook -- AHook)
+				local barTop : XH * OverlayPos * 1.02 + df.mvs * 0.425
+				local Bowl : BowlT Flat ac2 df.leftSB df.rightSB barTop 0 SmallArchDepthA SmallArchDepthB (stroke -- df.mvs)
+				include : dispiro
+					ArcC1.hookStartL
+					flat df.rightSB (XH - SmallArchDepthB)
+					curl df.rightSB Bowl.ArcEnd.yRB [heading Downward]
+				include : Bowl.shape.arcL
+				include : ArcC1.serifStartL
+
+		# Simplified impl of b
+		define [bfunc ac1 ac2 code name] : begin
+			create-glyph name code : glyph-proc
+				local df : DivFrame 1
+				local Bowl : BowlT ac1 ac2 df.leftSB df.rightSB XH 0 SmallArchDepthA SmallArchDepthB (stroke -- df.mvs)
+
+				include : Bowl.shape.arcR
+				include : VBar.l df.leftSB Bowl.ArcEnd.yLB Ascender df.mvs
+
+		# Simplified impl of c
+		define [cfunc ac1 ac2 code name] : begin
+			create-glyph name code : glyph-proc
+				local df : DivFrame 1
+				local Bowl : BowlT ac1 ac2 df.leftSB df.rightSB XH 0 SmallArchDepthA SmallArchDepthB (stroke -- df.mvs)
+
+				include : Bowl.shape.hookL
+				include : Bowl.ArcStart.serifStartR
+				include : Bowl.ArcEnd.serifEndR
+
+		# Simplified impl of single storey a
+		define [aafunc ac1 ac2 code name] : begin
+			create-glyph name code : glyph-proc
+				local df : DivFrame 1
+				local Bowl : BowlT ac1 ac2 df.leftSB df.rightSB XH 0 SmallArchDepthA SmallArchDepthB (stroke -- df.mvs)
+
+				include : Bowl.shape.arcL
+				include : VBar.r df.rightSB Bowl.ArcEnd.yRB Bowl.ArcStart.yRT df.mvs
+
+		# Simplified impl of e
+		define [efunc ac1 ac2 code name] : begin
+			create-glyph name code : glyph-proc
+				local df : DivFrame 1
+				# local ArcC1 : ac1 (df.leftSB + OX) (df.rightSB - OX) XH 0 (stroke -- df.mvs)
+				local ArcC2 : ac2 (df.leftSB + OX) (df.rightSB - OX) XH 0 (stroke -- df.mvs) (hook -- SHook)
+
+				local barBottom : XH * DesignParameters.eBarPos - (df.mvs / 2)
+				include : HBar.b
+					df.leftSB + [HSwToV : 0.5 * df.mvs] + OX
+					df.rightSB - [HSwToV : 0.5 * df.mvs] - OX
+					* barBottom
+					* df.mvs
+				local path : include : dispiro
+					widths.lhs df.mvs
+					flat (df.rightSB - OX) barBottom [heading Upward]
+					curl (df.rightSB - OX) (XH - SmallArchDepthB)
+					arch.lhs XH (sw -- df.mvs)
+					flatside.ld df.leftSB 0 XH SmallArchDepthA SmallArchDepthB 0
+					ArcC2.hookEndR
+				# include : ArcC1.serifStartR
+				include : ArcC2.serifEndR
+
+		# Simplified impl of ь (?)
+		define [bbfunc ac1 ac2 code name] : begin
+			create-glyph name code : glyph-proc
+				local df : DivFrame 1
+				define YeriBarPos 0.55
+				local yArcTop : XH * YeriBarPos + df.mvs / 2
+
+				local Bowl : BowlT ac1 ac2 df.leftSB df.rightSB yArcTop 0 SmallArchDepthA SmallArchDepthB (stroke -- df.mvs)
+
+				include : Bowl.shape.arcR
+				include : VBar.l df.leftSB Bowl.ArcEnd.yLB XH df.mvs
+
+		# mask test based on double storey a shape
```

Comment:
Masks should cover the entire internal of the shoulder/bowl/couner.

Reply:
Also you need to specify the _offset_ of masks. In our implementation masks typically have a tiny offset to avoid handling of occurrent outlines.